client/src/layouts/Plain/index.js

```diff
@@ -14,10 +14,10 @@ const useStyles = makeStyles((theme) => ({
   root: {
     display: "flex",
     flexDirection: "column",
-    paddingTop: 64,
+    paddingTop: 60,
```

Comment:
I think this padding is adjusted with header height. So we need to consider this.